corehq/apps/auditcare/utils/migration.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime, timedelta
+
+from django.core.cache import cache
+
+from dimagi.utils.dates import force_to_datetime
+
+from corehq.apps.auditcare.models import AuditcareMigrationMeta
+
+INITIAL_START_DATE = datetime(2013, 1, 1)
+
+
+class AuditCareMigrationUtil():
+
+    def __init__(self):
+        self.start_key = "auditcare_migration_2021_next_batch_time"
+        self.start_lock_key = "auditcare_migration_batch_lock"
+
+    def get_next_batch_start(self):
+        return cache.get(self.start_key)
+
+    def generate_batches(self, worker_count, batch_by):
+        batches = []
+        with cache.lock(self.start_lock_key, timeout=10):
+            start_datetime = self.get_next_batch_start()
+            if not start_datetime:
+                # for the first call
+                start_datetime = INITIAL_START_DATE
+
+            start_time = _get_formatted_start_time(start_datetime, batch_by)
+            end_time = None
+
+            for index in range(worker_count):
+                end_time = _get_end_time(start_time, batch_by)
+                batches.append([start_time, end_time])
+                start_time = end_time
+            self.set_next_batch_start(end_time)
+            if end_time > datetime.now():
+                print("Migration successfully done")
+                exit(1)
```

Comment:
I think this should `0` since `1` indicates an error.
```suggestion
                exit(0)
```

Reply:
Yep. It should be 0.